purchases/src/main/kotlin/com/revenuecat/purchases/common/networking/HTTPResult.kt

```diff
@@ -21,6 +22,7 @@ internal data class HTTPResult(
     val requestDate: Date?,
     val verificationResult: VerificationResult,
     val isLoadShedderResponse: Boolean? = null,
+    val isFallbackURL: Boolean? = null,
```

Comment:
Similar question as in the other PR. Should it be nullable or can it just be `Boolean` (without "?")?